crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -114,10 +119,10 @@ pub struct Task<TxPool, D> {
     continue_on_error: bool,
 }
 
-impl<TxPool, D> Task<TxPool, D>
+impl<'a, TxPool, D> Task<TxPool, D>
 where
     TxPool: ports::worker::TxPool,
-    D: ports::worker::OffChainDatabase,
+    D: ports::worker::OffChainDatabase + 'a,
```

Comment:
Hmm, I don't see why we need a lifetime here=)

Reply:
50d53d1424